packages/fuel-indexer-macros/src/indexer.rs

```diff
@@ -377,6 +377,21 @@ fn process_fn_items(
                 }
             }
 
+            fn compute_message_id(&self, sender: &Address, recipient: &Address, nonce: Bytes32, amount: Word, data: &[u8]) -> MessageId {
```

Comment:
If this is a helper function can we move it to helpers.rs ?

Reply:
As far as I can tell, this has to reside on the decoder struct as it uses the values from the receipt fields to calculate the ID for the ABI type. I can create a helper function that returns a `TokenStream` that contains the function and inline it, if you like.